starport/templates/testutil/register.go

```diff
@@ -25,5 +29,8 @@ func Register(ctx *plush.Context, gen *genny.Generator) error {
 	if !ctx.Has(modulePathKey) {
 		return fmt.Errorf("ctx is missing value for the ket %s", modulePathKey)
 	}
+	if _, err := os.Stat(testUtilDir); !os.IsNotExist(err) {
```

Comment:
Scaffolding commands accepts a `--path` option so, we cannot directly stat `testutil`.

Actually, seems like [Register](https://github.com/tendermint/starport/blob/fbce102236488b7e8ff700b77dd839d9d62dde6f/starport/templates/typed/typed.go#L56) never checked if it returns an error which we also should fix. Once we do that, all type related scaffolding commands will start to fail because the `testutil` dir will always be exists if user hasn't deleted.

It doesn't needs to be in this PR but we can actually move `testutil` to tendermint/spm.